frontend/screens/SignUpScreen.js

```diff
@@ -0,0 +1,343 @@
+import Airtable from "airtable";
+import { Notifications } from "expo";
+import Constants from "expo-constants";
+import * as Permissions from "expo-permissions";
+import * as WebBrowser from "expo-web-browser";
+import React from "react";
+import { AsyncStorage, Button, Keyboard, StyleSheet, Text, TextInput, View } from "react-native";
+import validatejs from "validate.js";
+
+// I abstracted portions of the validation flow into these files 
+// but there's a weird bug "https://github.com/facebook/react-native/issues/4968"
+// so I put it all in this one file.
+// import { MonoText } from 'screens/signup/textfield.jsx'
+// import validation from  'screens/signup/validation'
+// import validate from  'screens/signup/validation_wrapper'
```

Comment:
Sounds good - could you start comments for future work with "TODO: @<github username>"? (Will communicate to this everyone going forward!) It's easier to find things we should go back and look at by searching in the codebase!